hawkular-wildfly-monitor/src/main/resources/module/main/module.xml

```diff
@@ -34,7 +34,25 @@
     <resource-root path="jackson-annotations.jar"/>
     <resource-root path="jackson-core.jar"/>
     <resource-root path="jackson-databind.jar"/>
+
+    <!-- These are pulled in due to hawkular-inventory-json-helper.jar.
+         It is probable that we really don't need all of these,
+         but without knowing which are really used by hawkular-inventory-json-helper
+         it is best we include all of them. -->
```

Comment:
I am not happy about that. 
We should try to keep the agent as small as possible.